setup.cfg

```diff
@@ -27,23 +27,21 @@ python_requires = >=3.7
 install_requires = 
     astropy
     numpy
-    sregion>=1.0
+    sregion
 packages = find_namespace:
 
 [options.package_data]
-data = 
+grizli.data =
+    *.yml
+    *.txt
+    *.fits
+    *.fits.gz
```

Comment:
There also seems to be a `.pkl` file in `grizli/data` as well (although the commit related to that indicates that it is a junk file; c7d4035).
Is that why we are excluding that?

Reply:
That was my though.  pickle files should never be in a build distribution because they are not platform independent (or even version independent).  Thanks for catching this and pointing it out.

It would be good to find out what this pickle file is there for and get it into a better format.

Reply:
Created an issue for this (#9) so that we can come back to this later.

Reply:
🤯 

Thanks for filing the issue.  Is there a more general way to save a scikit-learn model than as a pickle file?  Can it be serialized using another format with an old version of scikit-learn?

https://scikit-learn.org/stable/model_persistence.html

Reply:
> However it was saved with a circa 2018 version of `sklearn` and cannot be read out with the current one.
That explains why I wasn't able to read it in at all!

Is there any way to see the underlying code without having to execute it? Maybe we can port the model to the newer API version.

Reply:
We need scikit-learn==0.20.4 to open it I think (https://github.com/HesamKorki/Pedestrian-detection/issues/1)

Reply:
Yup, what you suggested works!
Notably, `scikit-learn` (helpfully) warned during the unpickling that the classifier model was pickled using v0.19.1, so we should stick to using that particular version for the time being.

For reference, here's the code which opened the pickle file:

#### Create a `conda` environment
```shell
conda create -n old-sklearn scikit-learn==0.19.1
conda activate old-sklearn
```

#### Open `sep_catalog_junk.pkl`
```python
from sklearn.externals.joblib import numpy_pickle
numpy_pickle.load('sep_catalog_junk.pkl')
```

#### Output
This produces an `SVC()` object like so:
```
SVC(C=1.0, cache_size=200, class_weight=None, coef0=0.0,
  decision_function_shape='ovr', degree=7, gamma='auto', kernel='rbf',
  max_iter=-1, probability=True, random_state=None, shrinking=True,
  tol=0.0001, verbose=False)
```

Reply:
Cool!  Can the parameters of this `SVC()` object then be used to instantiate something in a current version of scikit-learn?  And if so, then that current version be serialized in a standard way?

Reply:
@BSGalvan is it possible to save this as a `*.joblib` file in that version of `scikit-learn`? Maybe such a file would be compatible with the current `joblib` library?

Reply:
@ivastar it is possible to save this as a `*.joblib` file within `scikit-learn v0.19.1`, however it is _not_ compatible with the current `joblib` because `scikit-learn` was reorganized [here](https://github.com/scikit-learn/scikit-learn/commit/5c5635feb75056f15fa77fdd5b19e655751e6de9#diff-931d533c34e17c17f795e73544bfbf993f1d4929c0599ceb02716dd01456159a), so that `sklearn.svm.classes.SVC()` became `sklearn.svm.SVC()`.

Trying to load a v0.19.1-pickled file with the current `joblib` thus raises a `ModuleNotFoundError`.

We can instead do what @jdavies-st suggested, and try to `pickle` the parameters of the `SVC()` object in a current version of `scikit-learn`, although I've not been able to find a way to access the _fitted_ model's parameters so that we may pickle them.

Then again this _will still_ cause issues in the future, since compatibility across (Python, library) versions is guaranteed by [neither `pickle`](https://scikit-learn.org/dev/model_persistence.html#security-maintainability-limitations), [nor `joblib`](https://joblib.readthedocs.io/en/latest/persistence.html#compatibility-across-python-versions) (which uses `pickle` under the hood anyway).

We may need to think about the solutions given [here](https://scikit-learn.org/dev/model_persistence.html#interoperable-formats) . . .